sys/include/net/gcoap.h

```diff
@@ -29,6 +29,14 @@
  * provides functions to generate and send an observe notification that are
  * similar to the functions to send a client request.
  *
+ * *Contents*
+ *
+ * - Server Operation
+ * - Client Operation
+ * - Observe Server Operation
+ * - Implementation Notes
+ * - Implementation Status
```

Comment:
[[TOC]](https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_toc) or [\tableofcontents](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdtableofcontents) should have the same effect.